modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7299,7 +7298,7 @@ feature.flag.LPS-196724.description=Users can add read-only fields to object def
 feature.flag.LPS-196724.title=Auto-Increment Field Type for Objects
 feature.flag.LPS-196768.description=Users can combine navigation filters, search by folder, and filter search results for web content. Improve search and filtering performance.
 feature.flag.LPS-196768.title=Improved Search and Filtering for Web Content
-feature.flag.LPS-196935.description=Dynamic Data Lists is deprecated and replaced by Object. DDL will be removed in 2024. Please migrate your data to Object before then.
+feature.flag.LPS-196935.description=Dynamic Data Lists is deprecated and replaced by Object. DDL will be removed in 2024. Please migrate your data to Object before thenddl-will-be-removed-in-2024
```

Comment:
Hey @diogomelo09, I think you added a text here by mistake when removing the language key. Could you please fix this?

Reply:
done :heavy_check_mark: